src/core/CMakeLists.txt

```
@@ -10,8 +10,7 @@ add_library(ostree-tui_core commit.cpp
                             manager.hpp
                             OSTreeTUI.cpp
                             OSTreeTUI.hpp
```

```suggestion
                            OSTreeTUI.hpp)
```

Or indent the `)` correctly.